reana_commons/api_client.py

```diff
@@ -99,6 +99,9 @@ def submit(  # noqa: C901
         compute_backend=None,
         kerberos=False,
         kubernetes_uid=None,
+        kubernetes_cpu_request=None,
```

Comment:
Since this is a minor patchlevel release, and since somebody may be using Python API to submit workflows, perhaps we should add the new optional arguments at the end? Ideally, they would be close together, but let's check how [RECAST](https://github.com/recast-hep) uses the API to make sure that we don't break their submission workflow by rearranging argument order.

(The safest option would be just to add them at the end, even though it's not pretty...)